PacificPower/readPP.js

```diff
@@ -70,7 +70,30 @@ let continueVarMonthly = 0;
 let page = "";
 let pp_meter_id = "";
 
+let pp_meter_exclude_yes = [];
+let pp_meter_exclude_no = [];
+let pp_meter_exclude_not_found = [];
+
 (async () => {
+  // fetch the meter exclusion list
+  const pp_meter_exclusion_list = await axios({
+    method: "get",
+    url: `${process.env.DASHBOARD_API}/ppexclude`,
+  })
+    .then((res) => {
+      console.log(
+        `RESPONSE: ${res.status}, TEXT: ${
```

Comment:
Should a non 200 status code throw an error? (I think axios get always returns 200 if the request succeeded, and not 302 etc)